### ISSUE TITLE
jdk11 docker build fails with missing X11 headers

### ISSUE THREAD

**Issue by lumpfish** [eclipse/openj9#6767]
I am attempting to build jdk11 following the instructions at https://www.eclipse.org/openj9/oj9_build.html#version-11

The configure command
```
bash ./configure --with-freemarker-jar=/root/freemarker.jar
```
fails with the message
```
configure: error: Could not find all X11 headers (shape.h Xrender.h Xrander.h XTest.h Intrinsic.h). You might be able to fix this by running 'sudo apt-get install libx11-dev libxext-dev libxrender-dev libxrandr-dev libxtst-dev libxt-dev'.
configure exiting with result code 1
```
Running the suggested command gives:
```
apt-get install libx11-dev libxext-dev libxrender-dev libxrandr-dev libxtst-dev libxt-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package libxrandr-dev
```
Excluding the libxrandr-dev package completes OK but the configure then fails in the same way.

**Comment by DanHeidinga** [comment]
Have you run `apt-get update` prior to running the `apt-get install` command?

**Comment by lumpfish** [comment]
```
apt-get update
```
followed by
```
apt-get install libx11-dev libxext-dev libxrender-dev libxrandr-dev libxtst-dev libxt-dev
```
fixed the issue.  Thank you.

Does this mean that the Dockerfile which the instructions ask the user to download and run should be updated?  Or at least that the instructions should be updated to add this step?

**Comment by keithc-ca** [comment]
I will make sure this problem is addressed by the work I'm doing for #5781.
